server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -455,20 +466,46 @@ export function configureWebSocketServices(
 						socket.emit("nack", "", [nackMessage]);
 						return;
 					}
-					contentBatches.forEach((contentBatch) => {
-						const contents = Array.isArray(contentBatch)
-							? contentBatch
-							: [contentBatch];
 
-						for (const content of contents) {
-							const signalMessage: ISignalMessage = {
-								clientId,
-								content,
-							};
+					if (supportedFeaturesMap.get(clientId)?.submit_signals_v2) {
+						for (const signal of contentBatches) {
+							if (isSentSignalMessage(signal)) {
+								const signalMessage: ISignalMessage = {
+									clientId,
+									...signal,
```

Comment:
`...signal` persists `targetClientId` thru the result, which is what we want. Currently the protocol-definitions used by server has the old versions of `ISignalMessageBase` and `ISentSignalMessage` that only has `targetClientId` in `ISentSignalMessage`.
How do we get protocol-definitions version updated? @znewton?

Reply:
They are now bumped to the updated protocol definitions

Reply:
Should be able to just update the package dependency, but I'm not sure how far behind the service is. Might be worth pulling that upgrade into a separate PR

Reply:
Why did I think it was behind? 3.2.0 is sufficient.